src/app.ts

```diff
@@ -72,28 +65,6 @@ gatewayApp.get('/', (_req: Request, res: Response) => {
   res.status(200).json({ status: 'ok' });
 });
 
-interface ConnectorsResponse {
```

Comment:
this seems like a cleanup change unrelated to injective?

Reply:
No, it's not.
It's not needed as well. `ConnectorsRoutes.router` does the same and is used by the client.
So we removed it.
However, I can undo it if there's a strict requirement.